docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -626,6 +626,8 @@ The following table properties are available for use:
     - Set the checkpoint interval in seconds.
   * - ``change_data_feed_enabled``
     - Enables storing change data feed entries.
+  * - ``column_mapping_mode``
+    - Column mapping mode. Possible values are ``ID``, ``NAME`` and ``NONE``. Defaults to ``NONE``.
```

Comment:
nit: Use a list as done for `delta.compression-codec`: 

https://github.com/trinodb/trino/blob/fd7fcb2dcdeeae014133127d02d54e1c80b4de8e/docs/src/main/sphinx/connector/delta-lake.rst?plain=1#L99-L106